rfremoterunner/rf_client.py

```diff
@@ -0,0 +1,168 @@
+import os
+import logging
+from six import StringIO
+import six.moves.xmlrpc_client as xmlrpc_client
+from robot.writer import DataFileWriter
+from robot.running import TestSuiteBuilder
+from robot.parsing.model import ResourceFile, TestCaseFile
+from robot.libraries import STDLIBS
+from robot.utils.robotpath import find_file
+from rfremoterunner.utils import normalize_xmlrpc_address, read_file_from_disk, calculate_ts_parent_path
+
+
+logger = logging.getLogger('rfremoterunner.executor')

```

Comment:
this has the same logger name as executor.py, presume this should be different

Reply:
I've tidied up the logging